### PR TITLE
docs: Remove prepare from contributing guide

### PR DESCRIPTION
**Pull request by moofish32**
By submitting this pull request, I confirm that my contribution is made under the terms of the Apache-2.0 license.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -108,7 +108,7 @@ Or:
 
 After you've bootstrapped the repo, you would probably want to work on individual packages.
 
-All packages in the repo have a two useful scripts: `prepare` and `watch`. In order to execute
```
Comment (eladb) [inline CONTRIBUTING.md]: "prepare" was actually renamed to "build", but I wouldn't remove it

Comment (moofish32) [reply CONTRIBUTING.md]: I'm a little confused when I run `prepare` in `@aws-ckd/aws-ec2` I get:

```
lr prepare
info cli using local version of lerna
lerna notice cli v3.4.1
lerna info filter [ '@aws-cdk/aws-ec2' ]
lerna success run No packages found with the lifecycle script 'prepare'
```

So what would a new contributor do with `prepare`?

**Comment by rix0rrr** [comment]
I'd be happy to merge but this has a conflict now. Is it perhaps not necessary anymore?

**Comment by moofish32** [comment]
agree issue is addressed let's close

**Pull request closed without merging**